docs/source/vllm_integration.md

```diff
@@ -1,14 +1,115 @@
 # vLLM Integration
+If you are here to learn about how to use vLLM with TRL, you are in the right place. This document will guide you through the process of using vLLM with TRL for faster generation in online methods like GRPO and Online DPO. We first summerize a tl;dr on how to use vLLM with TRL, and then we will go into the details of how it works under the hood. Let's go! 🔥
 
-<Tip warning={true}>
+## 🫠🚀 How can I use vLLM with TRL to make things go faster? TL;DR
+First run the server by; (This example allocate 4 GPUs for vLLM generation)
+```sh
+trl vllm-serve --model Qwen/Qwen2.5-7B --tensor-parallel-size 2 --data-parallel-size 2
+```  
+Then, run the training script by passing `use_vllm=True` in the training arguments (this example allocate 4 GPUs for training) by;
+  
+```sh
+CUDA_VISIBLE_DEVICES=4,5,6,7 accelerate launch train.py
+```  
+
+Sample of a simple `train.py` script:
```

Comment:
I suggest moving this up before the train command. Also, I would indicate whether you are using FSDP or ZeRO-2 here since I don't think a 7B model fits in DDP